spec/acceptance/suites/default/00_default_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper_acceptance'
+
+test_name '<%= metadata.name %> class'
```

Comment:
looks like an erb <%= metadata.name *>?

Reply:
None of the  skeleton's boilerplate  tests make much sense in a module without any classes.

I've removed them for now.